src/pytorch_lightning/loggers/wandb.py

```diff
@@ -223,7 +223,7 @@ def __init__(self, *args, **kwarg):
 
     Args:
         name: Display name for the run.
-        save_dir: Path where data is saved (wandb dir by default).
+        save_dir: Path where data is saved.
```

Comment:
As discussed in https://github.com/Lightning-AI/lightning/issues/12036#issuecomment-1048268401, It's not a good idea to use [` wandb.Run.dir`](https://docs.wandb.ai/ref/python/run). In fact, currently WandbLogger uses the  `dir` pass to [`wandb.init`](https://docs.wandb.ai/ref/python/init) as `save_dir`, which is the same level of `wandb.Run.dir`, so I removed the confusing documentation.